packages/driver/cypress/support/utils.js

```diff
@@ -107,7 +107,10 @@ const getAllFn = (...aliases) => {
   )
 }
 
-const shouldWithTimeout = (cb, timeout = 250) => {
+// FIXME: currently increasing the timeout here from 250ms to 1 second
```

Comment:
Should we log an issue for this?

Reply:
We should. @mschile logged #21361 for some of the proxy issues happening in electron from this branch. I'll go ahead and create and issue and update the thread here

Reply:
issue logged in #21375